css/style.css

```diff
@@ -791,6 +791,10 @@ body button {
   line-height: 1.3;
 }
 
+.work-card__h2 {
```

Comment:
`work-card__h2` is not best solution, because in html we can will change node. Let's name this `work-card__category`